pom.xml

```diff
@@ -2574,7 +2574,8 @@
         <plugin>
           <groupId>net.alchim31.maven</groupId>
           <artifactId>scala-maven-plugin</artifactId>
-          <version>4.5.3</version>
+          <!-- SPARK-36547: Please don't upgrade the version below, otherwise there will be an error on building Hadoop 2.7 package -->
+          <version>4.3.0</version>
```

Comment:
My bad. I forgot the history. :(

Reply:
No worries .. It was only able to reproduce in some specific environments. I don't think there was anything wrong when we upgrade.

Reply:
In fact, ideally it should be investigated why it fails in the specific environments .. but I am fine to unblock the release for now.